components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/profiler/Profiler.kt

```diff
@@ -63,4 +64,21 @@ class Profiler(
     override fun addMarker(markerName: String) {
         runtime.profilerController.addMarker(markerName)
     }
+
+    override fun startProfiler(filters: Array<String>, features: Array<String>) {
```

Comment:
nitpick (non-blocking): I think `List<String>` is more natural in java and kotlin. Perhaps we should take a List<String> and convert it to an array before passing it to the profiler controller?

Reply:
We could, this would mean changes in Fenix too (albeit small)